examples/machine_translation/utils.py

```diff
@@ -0,0 +1,62 @@
+import itertools
```

Comment:
Are there any generic building blocks that we could add to the library to support other pipeline?

Reply:
`pad_chars` and `pad_words` would essentially be useful for other pipeline for sure, but for `pad_chars` not really sure how often people are actually using it compares to subword/word level. the rest (`epoch_time`, `count_parameters`, and `seed_everything`) are typically available on training framework such as pytorch-lightning, ignite, etc, I don't think we need to re-implement them in torchtext, what do you think?